Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -126,6 +126,21 @@ public Element build(final CallbackProperties properties) throws SAMLAssertionBu
             final Subject subject = createSubject(properties, certificate, publicKey);
             assertion.setSubject(subject);
 
+            // add conditions statement
+            DateTime beginValidTime = new DateTime();
```

Comment:
I'm assuming this will be updated to include the new assertion values you're adding with common types?